core/src/time.rs

```diff
@@ -105,21 +63,17 @@ pub fn format_rfc3339(t: DateTime) -> String {
 /// - `2022-03-01T08:12:34+00:00`
 /// - `2022-03-01T08:12:34.00+00:00`
 pub fn parse_rfc3339(s: &str) -> crate::Result<DateTime> {
```

Comment:
jiff's Timestamp from_str actually accepts more inputs than exactly RFC3339. Not sure if we should change the method name or just be OK with this.

Reply:
Do you have an example for this?

Reply:
You may read https://docs.rs/jiff/latest/jiff/fmt/temporal/index.html

Reply:
There's a number of them:

```rust
    assert!("2025-09-22T17Z".parse::<jiff::Timestamp>().is_ok());
    assert!("2025-09-22T17:30Z".parse::<jiff::Timestamp>().is_ok());
    assert!("2025-09-22T17:30:00-04".parse::<jiff::Timestamp>().is_ok());
    assert!("20250922T173000-04".parse::<jiff::Timestamp>().is_ok());
```

Jiff uses [Temporal's ISO 8601](https://tc39.es/proposal-temporal/#sec-temporal-iso8601grammar) format. It's a blend of RFC 3339, RFC 9557 and ISO 8601. You can read [more about it here](https://docs.rs/jiff/latest/jiff/fmt/temporal/index.html).

In theory Jiff could provide a "strict RFC 3339" parser, but it's not clear to me how useful that is.

Reply:
Oh I see. I think the current behavior is fine, and I actually think it's better to let `parse_rfc3339` can accept more valid inputs.